### ISSUE TITLE
If I try to "fine-tune" a model on a dataset where the number of classes are not 80 (coco) then am I essentially training from scratch?

### ISSUE THREAD

**Issue by aymuos15** [ultralytics/ultralytics#19162]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Whenever I try to fine-tune, the scores start from 0 when training.

I am confused as to what the benefit is exactly of the pre-trained model given that I see this line: Transferred 448/499 items from pretrained weights. 

When someone fine-tune something like ResNet for classification, you change just the linear layer. 

Thanks!

**Comment by Skillnoob** [comment]
The pretrained model is used to act as a base model to train off, it usually results in better accuracy than training with `yolo11n.yaml` for example. If you want to limit the layers that get trained you can use the `freeze` argument, see [train settings](https://docs.ultralytics.com/modes/train/#train-settings).

**Comment by aymuos15** [comment]
Right. Thank you. 

Could you please also direct me to the code (or just tell me haha) about which layers exactly are removed when this statement comes: `Transferred 448/499 items from pretrained weights.`

**Comment by Skillnoob** [comment]
Only the [`Detect`](https://github.com/ultralytics/ultralytics/blob/9d8c2fe3c7a2701a9cdc769b4b8786738a5b99d0/ultralytics/cfg/models/11/yolo11.yaml#L50) layer (in the case of object detection) is not transferred since you have a different class count. All other layers are transferred.

**Comment by aymuos15** [comment]
Thank you very much.

**Comment by glenn-jocher** [comment]
@aymuos15 you're welcome! The layers not transferred are specifically the final detection head layers (like `Detect`) that depend on class count `nc`. All backbone/neck layers retain pretrained weights. For code details, see the `parse_model()` function in the [tasks.py](https://docs.ultralytics.com/reference/nn/tasks/#ultralytics.nn.tasks.parse_model) source that handles layer initialization when class counts change.
